lib/install/validate/storage.go

```diff
@@ -58,18 +62,56 @@ func (v *Validator) storage(ctx context.Context, input *data.Data, conf *config.
 	}
 
 	// TODO: add volume locations
-	for label, volDSpath := range input.VolumeLocations {
-		dsURL, _, err := v.DatastoreHelper(ctx, volDSpath, label, "--volume-store")
-		v.NoteIssue(err)
+	for label, volPath := range input.VolumeLocations {
+		nfsURL, nfsErr := NFSHelper(volPath)
+		if nfsURL != nil && nfsErr == nil {
+			conf.VolumeLocations[label] = nfsURL
+			continue
+		}
+
+		dsURL, _, dsErr := v.DatastoreHelper(ctx, volPath, label, "--volume-store")
+
+		// if both validations return error then the input cannot be used as a volumestore.
+		if nfsErr != nil && dsErr != nil {
+			v.NoteIssue(nfsErr)
+			v.NoteIssue(dsErr)
+		}
+
 		if dsURL != nil {
 			conf.VolumeLocations[label] = dsURL
 		}
 	}
 }
 
+// NFSHelper is used to construct a potential nfs target. If a valid NFS target cannot be contsructed a nil pointer will be returned.
+func NFSHelper(rawPath string) (*url.URL, error) {
+	// NOTE: issues from this function will need to be noted at the end of the datastore validation attempt. since we have multiple acceptable input formats now.
```

Comment:
But the `NOTE` portion shows up as a different color in most editors which means people are more likely to read them ;-)